src/reconstruction/refinement/removeFieldEntriesForType.m

```diff
@@ -76,7 +76,11 @@
     end
     %update the rules fields.
     if isfield(model,'rules') %Rely on rules first  
-        %However, we first normalize the rules.
+        rulesFieldOk = verifyModel(model,'simpleCheck',true,'restrictToFields',{'rules'}, 'silentCheck', true);
+        if ~rulesFieldOk
+            error('Rules field does not satisfy the field definitions. Please check that it satisfies the definitions given <a href="https://github.com/opencobra/cobratoolbox/blob/master/docs/source/notes/COBRAModelFields.md">here</a>');
```

Comment:
May you implement the error message using [hyperlink.m](https://github.com/opencobra/cobratoolbox/blob/master/src/base/hyperlink.m) such that the error message is correctly displayed when MATLAB is executed in console mode?

Reply:
Done.